Toast/style.css

```diff
@@ -0,0 +1,9 @@
+.toast {
+  padding: 1rem;
+  font-size: 1.2rem;
+  width: 400px;
+  background-color: white;
+  text-align: center;
+  border-radius: 5px;
+  border: 1px solid #4caf50;
+}
```

Comment:
Suggestion, if you could create toast relative to the viewport, & maybe also show a demo. It would be great!

Reply:
Thank you so much for the suggestion. I'll do